tests/test_cli/test_main.py

```
@@ -0,0 +1,200 @@
+    assert "--download-mode" in result.output
+    assert "--years" in result.output
+    assert "--quarters" in result.output
+
+def test_cli_missing_config(cli_runner):
+    """Test CLI with missing config file."""
+    result = cli_runner.invoke(main, ["--config-file", "nonexistent.yaml"])
+    assert result.exit_code != 0
+    assert "Error:" in result.output
+    assert "does not exist" in result.output
+
+def test_cli_invalid_config_format(cli_runner, tmp_path):
```

Ruff (0.8.2)

Missing return type annotation for public function `test_cli_invalid_config_format`

Add return type annotation: `None`

(ANN201)